src/test/macros.rs

```diff
@@ -20,7 +20,7 @@ macro_rules! uint {
     ($value:expr) => {
         $value[2..]
             .parse::<web3::types::U256>()
-            .expect("valid address")
+            .expect("invalid address")
```

Comment:
why the switch from valid to invalid without any logical change? Was this a typo in the first place?

Reply:
Yes shhhhh...